src/commands/skip.rs

```diff
@@ -0,0 +1,66 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use serenity::builder::CreateApplicationCommand;
+use serenity::{async_trait, model::prelude::command::CommandOptionType::Integer};
+
+use crate::command_context::CommandContext;
+use crate::store::guild_action::play_next_track::PlayNextTrack;
+use crate::store::guild_action::remove::Remove;
+
+use super::{CommandDataOptionUtil, JoovyCommand, JoovyCommands};
+
+const FROM: &str = "from";
+const TO: &str = "to";
+
+#[derive(Default, Debug)]
+pub struct Skip;
+
+#[async_trait]
+impl JoovyCommand for Skip {
+    fn create_application_command<'a>(
+        &self,
+        command: &'a mut CreateApplicationCommand,
+    ) -> &'a mut CreateApplicationCommand {
+        command
+            .name(JoovyCommands::Skip(Skip).as_ref())
+            .description("Removes tracks from the queue. If from is not defined the current track will be skipped.")
+            .create_option(|option| {
+                option
+                    .name(FROM)
+                    .kind(Integer)
+                    .description("Start number to remove from (inclusive)")
+                    .min_int_value(0)
+                    .required(false)
+            })
+            .create_option(|option| {
+                option
+                    .name(TO)
+                    .kind(Integer)
+                    .description("End number to remove to (inclusive)")
```

Comment:
"Final track to remove" kanske. Btw tänker kanske att man kan göra så att om man endast anger ett from, så tar man bort alla som kommer efter? Bra om man råkar lägga till någon stor spellista eller något? Men man borde kanske ha en undo action då ifall man råkar ta bort mer än tänkt, alternativt om man lägger till ett dialog meddelande med yes and no buttons (joovy klarar väl knappar nu i och med att vi har med slash commands)

Reply:
Hmm, tror det blir lite mkt med en undo-action just nu. Tänker mig att vi kan göra lite mer avancerad playlist hantering när UI:t är på plats.

Reply:
Sounds good 👍